delivery/client.py

```diff
@@ -14,17 +14,41 @@
 import cnudie.retrieve
 import cnudie.util
 import delivery.model as dm
+import delivery.util as du
 import dso.model
 import http_requests
+import model
+import model.github
 
 
 logger = logging.getLogger(__name__)
+JWT_KEY = 'bearer_token'
 
 
 class DeliveryServiceRoutes:
     def __init__(self, base_url: str):
         self._base_url = base_url
 
+    def auth(self):
+        return ci.util.urljoin(
+            self._base_url,
+            'auth',
+        )
+
+    def auth_configs(self):
+        return ci.util.urljoin(
+            self._base_url,
+            'auth',
+            'configs',
+        )
+
+    def auth_public_signature_key(self):
+        return ci.util.urljoin(
+            self._base_url,
+            'auth',
+            'public-signature-key',
```

Comment:
Considering there might be different public-keys (1. upon rotation, 2. for different principals), I suggest to prepare for that by adding another path component. We might hardcode a default value (maybe just: `default`) for now, for example (`auth/public-key/default`). The route (on server-side) should interpret the last part as key-id.
We _might_ write (and read) key-id into JW-tokens (although I think this might be considered an insecure practise).

Reply:
As discussed, superseded by OpenID standard configuration.